sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/v2/Spark33XShims.scala

```diff
@@ -56,4 +57,24 @@ trait Spark33XShims extends Spark33XFileOptionsShims {
     new ParquetFilters(schema, pushDownDate, pushDownTimestamp, pushDownDecimal, pushDownStartWith,
       pushDownInFilterThreshold, caseSensitive, datetimeRebaseMode)
   }
+
+  override def getExprs: Map[Class[_ <: Expression], ExprRule[_ <: Expression]] = {
+    (super.getExprs.values ++ Seq(
+      GpuOverrides.expr[BitLength](
+        "The bit length of string data",
+        ExprChecks.unaryProject(TypeSig.INT, TypeSig.INT,
+                                TypeSig.STRING, TypeSig.STRING + TypeSig.BINARY),
+        (a, conf, p, r) => new UnaryExprMeta[BitLength](a, conf, p, r) {
+          override def convertToGpu(child: Expression): GpuExpression = GpuBitLength(child)
+        }),
+      GpuOverrides.expr[OctetLength](
```

Comment:
OctetLength has been in Spark since 2.3.0. Why are we only putting it into the 3.3 shim?